rskj-core/src/test/java/co/rsk/peg/federation/VoteFederationChangeTest.java

```diff
@@ -457,6 +454,20 @@ void voteCommitFederation_postRSKIP419_whenPendingFederationIsSet_shouldPerformC
         assertNoHandoverToNewFederation();
     }
 
+    private void assertIsTheExpectedFederation(Federation federation) {
+        Federation federationBuiltFromPendingFederation = pendingFederationToBe.buildFederation(
+            Instant.ofEpochSecond(RSK_EXECUTION_BLOCK_TIMESTAMP),
+            RSK_EXECUTION_BLOCK_NUMBER,
+            federationMainnetConstants,
+            activations
+        );
+
+        assertEquals(federationBuiltFromPendingFederation.getCreationTime(), federation.getCreationTime());
+        assertEquals(federationBuiltFromPendingFederation.getCreationBlockNumber(), federation.getCreationBlockNumber());
```

Comment:
```suggestion
        assertEquals(RSK_EXECUTION_BLOCK_TIMESTAMP, federation.getCreationTime().getEpochSecond());
        assertEquals(RSK_EXECUTION_BLOCK_NUMBER, federation.getCreationBlockNumber());
```
What do you think? I think it makes it more explicit hat we expect

Reply:
like! Thanks